### PR TITLE
Add Initial CODEOWNERS File

### PR DESCRIPTION
**Pull request by coreyja**
Add a CODEOWNERS file to help facilitate review of PRs

Currently it has the `community-moderators` Team as the default
CODEOWNERS for the entire repo, plus some other members from
Discord who wanted to be included.

This will likely change as the Docs site expands

### CODE REVIEW

**Review by jlafayette — APPROVED** [review]
Nice!

**Comment by 770grappenmaker** [comment]
Aren't you supposed to add all contributors as a codeowner, or am I missing something? What is the point of a codeowners file?

**Comment by jlafayette** [comment]
> Aren't you supposed to add all contributors as a codeowner, or am I missing something? 

Not sure about the decision for which contributors have been added as owners... looks like some people from the wiki thread were added but not everyone.

> What is the point of a codeowners file?

The CODEOWNERS file controls who needs to approve PRs before they can be merged.  If you are assigned as an owner then you get notified when a PR is opened.

I think using a team is nice because it does not require a PR to adjust the CODEOWNERS file when people join or leave the project.

One other benefit is that you can assign individuals or teams as owners of parts of the repo - I think this will be useful later on when the project gets bigger.  We could assign some people as experts to some specific pages or sections, and to modify those pages they would have to approve.  See [example-of-a-codeowners-file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file)

**Comment by DaBultz** [comment]
> Aren't you supposed to add all contributors as a codeowner, or am I missing something? What is the point of a codeowners file?

Codeowners is who should review and approve PRs, the people in the codeowners gets notified automatically when someone opens a PR and right now it's set to be 2 code owners must approve before it can get merged.

Read more from this link: 
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

> Not sure about the decision for which contributors have been added as owners... looks like some people from the wiki thread were added but not everyone.

Corey wrote about this on Discord, right now you just have to let him know.  But i think us 4 in this PR really will be the ones doing the most.

**Review by DaBultz — APPROVED** [review]
Looks good, not sure if anyone else would like to be added.

**Pull request merged**
